src/modules/BootStorage.rb

```diff
@@ -21,39 +21,39 @@
 require "storage"
 require "y2storage"
 require "bootloader/udev_mapping"
+require "bootloader/exceptions"
 
 module Yast
   class BootStorageClass < Module
     include Yast::Logger
-    using Y2Storage::Refinements::DevicegraphLists
-    using Y2Storage::Refinements::Disk
 
-    attr_accessor :mbr_disk
+    # @return [Y2Storage::Partitionable]
```

Comment:
Well, this is actually `Y2Storage::Disk` since you never use `Partition#partitionable` to access it, but always `Partition#disk`, which effectively discards any other partitionable type.